scriptgen

```diff
@@ -12,17 +12,18 @@ from subprocess import check_output
 node_list_file = 'node-list.json'
 output_directory = 'my-node-scripts'
 
-def format_script(address, ip, host, boot_node_ip, prvkey, vals):
+def format_script(address, ip, host, boot_node_ip, prvkey, vals, oauth):
     output = """\
 #!/usr/bin/env sh
 # Address: {0}
 # IP: {1}
-NODE_HOST="{2}" \\
-  BOOT_NODE_IP="{3}" \\
-  blockstanbulPrivateKey="{4}" \\
-  validators='{5}' \\
+export NODE_HOST="{2}" \\
+  export BOOT_NODE_IP="{3}" \\
+  export blockstanbulPrivateKey="{4}" \\
+  export validators='{5}' \\
+  export oauth_parameters='{6}' \\
```

Comment:
To answer your question about which variables should be added, I don't think that's the right direction to go in. We have a dictionary/JSON object for each node that we want to write a script for, and the environment is like a dictionary, so we should just translate from one to the other. 

I think the easiest way to do that is to change up the string formatting technique. Rather than having one format call, we can loop over the input JSON object and create one environment variable for every key that is not treated specially by scriptgen (like `host` or `ip`).

Something like:
```
>>> env = {"OAUTH_CLIENT_ID": "clientid", "OAUTH_ENABLED": "true"}
>>> xs = ["#!/usr/bin/env sh", "# Address: deadbeef"]
>>> xs.extend(["export {0}='{1}'".format(k, v) for k, v in env.iteritems()])
>>> xs.append("./strato --blockstanbul")
>>> print '\n'.join(xs)
#!/usr/bin/env sh
# Address: deadbeef
export OAUTH_CLIENT_ID='clientid'
export OAUTH_ENABLED='true'
./strato --blockstanbul
```

As an aside, with the `export` syntax they can all be aligned on the first column and the trailing backslashes are not necessary